testing/assertions/assertions.go

```diff
@@ -235,6 +236,27 @@ func LogsContain(loggerFn assertionLoggerFn, hook *test.Hook, want string, flag
 	}
 }
 
+// LogsDontHaveErrors checks whether there are no error logs in the output.
+func LogsDontHaveErrors(loggerFn assertionLoggerFn, hook *test.Hook, msg ...interface{}) {
```

Comment:
This is much more robust than what we have now, which is checking that no `Could not` logs exits. Not all error logs in the function have this format.